### DISCUSSION TITLE
[Crear flujo de trabajo]No se visualiza el display de la colección según una de las posibles dos condiciones

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#423] [4. Bug Reports] [answered]
Hola a todos

En la lista de campos adicionales de una task, no es posible agregar una colección ya que la lista disponible no muestra todas. Al parecer el máximo que muestra la lista es de 100 colecciones.

Ejemplo: si quiero agregar la colección fruta y pertenece a la colección 130, ésta no se visualizará dentro de la lista, por lo que debo hacerlo a través de la BD. Sin embargo al realizarlo por DB, en la vista de los campos adicionales del admin no visualizará el nombre, porque no se refleja en el listado de los 100, lo que puede provocar problemas si otra persona entra a configurar la máquina y pisa el campo adicional.

Adjunto SS desde el admin (FLUJO COMERCIAL - COTALKER) y en el admin el status 1 sale vacío.
![image](https://user-images.githubusercontent.com/61419471/134058901-27f4dafb-851c-440c-99b7-3778e441799f.png)

Adjunto code de la SM , donde se puede ver que tiene asignado los 5 status 
```
{ 
    "_id" : ObjectId("5f20a4d793bcab000896f80d"), 
    "hooks" : {
        "onCreate" : [

        ]
    }, 
    "dynamicPropertyTypes" : {
        "status1" : ObjectId("5e1f8ee7717f730c44b9850f"), 
        "status2" : ObjectId("5f75e3af5559c10008a21622"), 
        "status3" : ObjectId("5f75eefe676c6c000970a60d"), 
        "status4" : ObjectId("5f1b522630a90d0008ca8e03"), 
        "status5" : ObjectId("5fad4a4d88b0c000085e5754"), 
        "isActive" : [
            "status1", 
            "status2", 
            "status3", 
            "status4"
        ]
    }, 
    "isActive" : true, 
    "taskChannelType" : "bound", 
    "taskGroup" : ObjectId("5e66b96d82c82323e5e921c0"), 
    "name" : "Propuesta Comercial 2.0", 
    "code" : "propuesta_comercial_2_0", 
    "initialState" : ObjectId("5fa594b42f5af20008610474"), 
    "propertyType" : ObjectId("5e66b9cdb0ea306a316dd12f"), 
    "asset" : {
        "type" : "unique", 
        "propertyType" : ObjectId("5e66bb4882c82323e5e92f70"), 
        "property" : null
    }, 
    "company" : ObjectId("58fe83eda8b7259625b9b15d"), 
}
```

**Comment by jdurandpaz** [comment, marked as answer]
Añadí  el issue https://github.com/Cotalker/documentation/issues/425 para que podamos darle seguimiento. Muchísimas gracias!
